### ISSUE TITLE
Create native Android integration

### ISSUE THREAD

**Issue by Danielku15** [CoderLine/alphaTab#532]
Once we have the base code translated to Kotlin the next level is build proper native android controls for integration into apps.

**Comment by Danielku15** [comment]
I have thought of this but it does not make a lot sense. Here my thoughts on that

1) The platform specific integrations are typically quite slim from the amount of code they add to the repo. My goal is rather to port the C#/Kotlin specific parts over to TypeScript too, and then rely fully on transpiling. 

2) The main part of the library still is living here and is based on transpiling. Usually there is the base/core library and then the platform specific controls on top. I could pull the main repo as submodule and run transpiling there but this would then just be a shift of the CI/CD to a different repo, there is not any real gain on this but rather effort to get it running clean. 

3) The only thing I want to maybe move out, and also extend are the samples. They are not needed for development of alphaTab itself and rather hidden/confusing for integrators the way they are today. 


In short: A split into a polyrepo will just generate effort, and make the whole development chain more complicated.
